include/glow/Graph/Nodes.h

```diff
@@ -31,8 +31,12 @@ namespace glow {
 // Storage is the base class for Variables, which are bound to tensors, and
 // Placeholder nodes which are unbound.
 class Storage : public Node {
+  /// Specifies if the variable is trainable.
```

Comment:
Instead of "variable" maybe use something that is not tie to a class name, like the "value in that storage".

Reply:
Good catch.